app/vmctl/flags.go

```diff
@@ -95,6 +98,12 @@ var (
 			Usage: "Extra labels, that will be added to imported timeseries. In case of collision, label value defined by flag" +
 				"will have priority. Flag can be set multiple times, to add few additional labels.",
 		},
+		&cli.Int64Flag{
+			Name: vmRateLimit,
+			Usage: "Optional data transfer rate limit per worker in bytes per second.\n" +
```

Comment:
It looks a bit weird, that rate limiting calculates per worker. It can lead to questions and configuration errors.
 
I suggest to make it global, it's already supported concurrent access with mutex protection.

 WDYT?

Reply:
native api doesn't have such restriction.

Reply:
> native api doesn't have such restriction.

It is because we have only one connection for native api.

The reason why it has per-worker limit is that if there will be multiple workers (default is 2) there could be inconsistent limit usage among them.

Reply:
Please see updated